### DISCUSSION TITLE
Show contact at dashboard lists?

### DISCUSSION THREAD

**Discussion by Matthias84** [jonaswinkler/paperless-ng#566] [Feature Requests]
Hi, might it be possible to show also the documents contact within the lists we pin at the startpage dashboard? This will help to distinguish between (new) documents, as they might have only non intuitive filenames (e.g. timestamps, numberings, ...) but the matcher might already detected the contact.

As the contact labels might be long, we might use emojis or contact avatars (logo, photos, ...) for fast recognition?

**Comment by jonaswinkler** [comment]
Agreed, this is useful.

> As the contact labels might be long

I'd just put them in the same text body as the title; the line may wrap, but that's not too bad.
